cmd/thanos/flags.go

```diff
@@ -0,0 +1,75 @@
+package main
+
+import (
+	"fmt"
+
+	"github.com/go-kit/kit/log"
+	"github.com/go-kit/kit/log/level"
+	"github.com/improbable-eng/thanos/pkg/cluster"
+	"github.com/pkg/errors"
+	"github.com/prometheus/client_golang/prometheus"
+	"gopkg.in/alecthomas/kingpin.v2"
+)
+
+func regCommonServerFlags(cmd *kingpin.CmdClause) (*string, *string, func(log.Logger, *prometheus.Registry, bool, string, bool) (*cluster.Peer, error)) {
+	grpcBindAddr := cmd.Flag("grpc-address", "Listen ip:port address for gRPC endpoints (StoreAPI). Make sure this address is routable from other components if you use gossip, 'grpc-advertise-address' is empty and you require cross-node connection.").
```

Comment:
@bwplotka can you clarify what the help text is suppose to mean here (the sentence is confusing the intention). Thanks in advance.

Reply:
Which part you is confusing? (: The gossip one?  Basically it means that you should make sure this address is routable if those 3 things apply. 

It's not that clear, I agree - but we will remove gossip soon.